styles/components/_misc.scss

```diff
@@ -71,6 +71,10 @@
       opacity: 0.4;
       pointer-events: none;
 
+      .emoji {
+        margin-right: 0.3em;
```

Comment:
This is only in the rating strip, the margin right was removed from the other emoji instances.